app/components/rdfa-editor-container.js

```diff
@@ -35,12 +55,23 @@ export default class RdfaEditorContainerComponent extends Component {
    */
   @action
   setPrefix(element) {
-    element.setAttribute('prefix', this.prefixToAttrString(this.documentContext.prefix));
+    element.setAttribute(
+      'prefix',
```

Comment:
I know this code was already there, but I don't understand why `prefix` must be set through a modifier?